src/server_manager/web_app/app.ts

```diff
@@ -1060,6 +1085,13 @@ export class App {
       await this.selectedServer.removeAccessKeyDataLimit();
       this.appRoot.showNotification(this.appRoot.localize('saved'));
       this.refreshTransferStats(this.selectedServer, serverView);
+      await sleep(HATS_DIALOG_DELAY_MS);
+      if (!serverView.isAccessKeyDataLimitEnabled) {
+        // TODO(alalama): survey link
+        showHatsDialogOnce(
```

Comment:
There's an issue here: if the user responded to the survey when they enabled the feature, they will be prompted again here. We shouldn't prompt the user to answer the same survey twice.

Reply:
These are two distinct surveys - one for enabling and one for disabling.